lighthouse-core/test/computed/metrics/first-cpu-idle-test.js

```diff
@@ -66,7 +66,7 @@ describe('FirstInteractive computed artifact:', () => {
       pessimistic: Math.round(result.pessimisticEstimate.timeInMs),
     }).toMatchSnapshot();
     assert.equal(result.optimisticEstimate.nodeTimings.size, 19);
-    assert.equal(result.pessimisticEstimate.nodeTimings.size, 79);
+    assert.equal(result.pessimisticEstimate.nodeTimings.size, 80);
```

Comment:
curious and very unexpected that the optimistic estimate already included this somehow but the pessimistic didn't...

can we double check this? might point to a bug we're missing

Reply:
is it b/c cpu idle uses fmp like this?

```
    const fmpTimeInMs = extras.optimistic
      ? extras.fmpResult.optimisticEstimate.timeInMs
      : extras.fmpResult.pessimisticEstimate.timeInMs;
```

Reply:
That doesn't change the size of the `nodeTimings` map though, so I don't think it could affect this.

Reply:
No idea how to debug this.

Reply:
I'll take a look

Reply:
Ah it's not that the optimistic estimate already had it, it's just that it still gets filtered out by the optimistic estimate. 

https://github.com/GoogleChrome/lighthouse/blob/43678454b845b89e0caab2b3ce5f9fdda2492ea2/lighthouse-core/computed/metrics/lantern-interactive.js#L40-L43

This is slightly suboptimal but OKish from the simulation perspective because we're flooring on the first paint metrics if we end up too early and if it's that short it's unlikely to have a large impact to the last long task.

Reply:
cool, thanks for looking!